composer.json

```
@@ -2,6 +2,7 @@
   "name": "swedbank-pay/swedbank-pay-sdk-php",
   "type": "library",
   "description": "The Swedbank Pay SDK for PHP simplifies integrations against Swedbank Pay's API Platform by providing native PHP interface towards the REST API.",
+  "version": "3.0.0",
```

The version of the release branch is `3.4.1` while in `composer.json` it's `3.0.0`. The [previous release of the SDK](https://github.com/SwedbankPay/swedbank-pay-sdk-php/releases/tag/3.4.0) was versioned `3.4.0`. With all the changes in the SDK we need to do a major increment to `4.0.0`.

```suggestion
  "version": "4.0.0",
```